spec/test/collections/test_silva.py

```diff
@@ -0,0 +1,266 @@
+from yaml import safe_load
+from jsonschema import validate
```

Comment:
We have a custom validator that we use in the RE repo; it will take read the spec files from the supplied path, so you just need to change your tests to use that interface.
The validator is at `relation_engine_server/utils/json_validation.py`, and you can see some examples of running validation in the tests at `relation_engine_server/test/test_json_validation.py` or `spec/test/collections/test_djornl.py`.